debian/changelog

```diff
@@ -1,3 +1,55 @@
+ubuntu-advantage-tools (27.1) impish; urgency=medium
+
+  * d/control:
+    - specify debianutils min version
+  * d/changelog:
+    - fix lintian typos amend and redact incorrect 27.0 entry (GH: #1624)
+  * lintian:
+    - override ubuntu-advantage-pro wanted-by-target cloud-init
+    - override xenial specific errors
+    - rename package-specific overrides for pro vs tools
+  * New upstream release 27.1:
+    - apt-hook:
+      + avoid segfault when comparing null Apt file origin to esm
+        (LP: #1929123)
+      + avoid wrapping static message formats at 80 chars
+      + fix lintian warnings based on compiled binary (GH: #1626)
```

Comment:
I think the lintian entry should be moved to the lintian section

Reply:
@lucasmoura For this one, lintian pointed out the problem, but the fix was a change to the build flags we're using for the json hook, which is why I put it in the apt-hook section. Would it make more sense here if I change the message to "update go build flags based on lintian warnings"? Or do you still think it fits better in the lintian section?

Reply:
I think the updated message will be clearer. If we change it, I see no problem keeping this entry on that section

Reply:
Cool, will do!